pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -1344,7 +1346,7 @@ public byte[] GetAuditLogHash()
         // For hash creation
         private byte[] GetTimeStampBytesForHash()
         {
-            var bytes = BitConverter.GetBytes(TimeStamp.ToFileTime() / TimeSpan.TicksPerSecond); // We only serialize to hour:min:sec precision, lose the ticks so we can roundtrip
```

Comment:
If you are concerned about cross-platform stability then why to use a system-specific timestamp representation?

Reply:
You're right, that should really be in terms of unix epoch. Nice catch!

So now what? I guess cross platform code has to shift to windows file time before hashing. That's embarrassing.

Reply:
Yes, I'll have to reproduce file time calculation in Java. It's not that hard, but it introduces dependency on logic that we cannot control.

Reply:
Not sure I follow you there - what can't we control about our own hashing code? Or do you mean we can't control the windows file time epoch definition? That seems deeply unlikely to change.

Reply:
Yes, the later. I agree that it's stable enough so we don't need a fix here. It will be a bit awkward, but not a biggie.